### PR TITLE
Jetpack blocks: Remove 💥

### PR DESCRIPTION
**Pull request by sirreal**
Remove the `jetpack-blocks` package.

## Testing

Calypso isn't broken 🙌 
Any references to the package lying around to be cleaned up?

### CODE REVIEW

**Comment by ockham** [comment]
Wondering how soon we can actually merge this. Do we need it around until the Jetpack 7.2 release in case we need to fix a block (or would we do that from within the JP repo :scream:)? Or even longer so we can use it for a point release, if necessary? :slightly_frowning_face:

**Comment by simison** [comment]
This folder was there to use by calypso components/sections and blocks:

https://github.com/Automattic/wp-calypso/tree/deadf7401e0f5ada6f7f647b62b52f211349a005/client/assets

...those image files can prolly be moved back to calypso components and folder removed.

**Comment by simison** [comment]
>Wondering how soon we can actually merge this. Do we need it around until the Jetpack 7.2 release in case we need to fix a block (or would we do that from within the JP repo 😱)? Or even longer so we can use it for a point release, if necessary? 🙁

We have a branch where we can pull changes from Jetpack and do a release for 7.2 from there: https://github.com/Automattic/wp-calypso/tree/update/jetpack-blocks-72

**Comment by sirreal** [comment]
> Do we need it around until the Jetpack 7.2

I think we can move forward with this any time and let #31793 be the only home for blocks (bugfixes and be ported and released from there) until the `jetpack-blocks` package is deprecated.

File: packages/jetpack-blocks/.eslintrc.js [hunk 1]
```diff
@@ -1,23 +0,0 @@
-module.exports = {
```
Comment (sirreal) [inline packages/jetpack-blocks/.eslintrc.js]: Should we move this into `client/gutenberg/extensions`? It aligns with the babel override that's in place there.

Comment (simison) [reply packages/jetpack-blocks/.eslintrc.js]: That's a good idea.

Comment (scruffian) [reply packages/jetpack-blocks/.eslintrc.js]: I had to remove `import/no-extraneous-dependencies`, as there's no `package.json` in this dir.

**Comment by scruffian** [comment]
> This folder was there to use by calypso components/sections and blocks:
> 
> https://github.com/Automattic/wp-calypso/tree/deadf7401e0f5ada6f7f647b62b52f211349a005/client/assets
> 
> ...those image files can prolly be moved back to calypso components and folder removed.

@simison I don't see these files in Jetpack, so I think we should probably open another PR to move them there first?

**Comment by ockham** [comment]
CircleCI's `build-notifications` task is failing, since it can't find `mini-css-extract-plugin-with-rtl` (and I can repro locally). This is kinda weird, the plugin is a dependency of `@automattic/calypso-build`, which in turn is a dependency of our root `package.json` :thinking:

**Comment by ockham** [comment]
> > This folder was there to use by calypso components/sections and blocks:
> > https://github.com/Automattic/wp-calypso/tree/deadf7401e0f5ada6f7f647b62b52f211349a005/client/assets
> > ...those image files can prolly be moved back to calypso components and folder removed.
> 
> @simison I don't see these files in Jetpack, so I think we should probably open another PR to move them there first?

No, that's the point -- Jetpack doesn't need those. They were used by the `related-posts` block (example content) for a while, but now they aren't anymore. They're only used by Calypso, so we can (And should) move them back to the `assets/` dir, as Mikael is suggesting.

**Comment by scruffian** [comment]
> No, that's the point -- Jetpack doesn't need those. They were used by the `related-posts` block (example content) for a while, but now they aren't anymore. They're only used by Calypso, so we can (And should) move them back to the `assets/` dir, as Mikael is suggesting.

Ok, on it :)

File: packages/jetpack-blocks/.eslintrc.js (hunk 1 above)
Comment (simison) [reply packages/jetpack-blocks/.eslintrc.js]: done in https://github.com/Automattic/wp-calypso/pull/31759/commits/a5b305ed2171cae85165a284fa56e1a456c3dabd

**Comment by ockham** [comment]
> > No, that's the point -- Jetpack doesn't need those. They were used by the `related-posts` block (example content) for a while, but now they aren't anymore. They're only used by Calypso, so we can (And should) move them back to the `assets/` dir, as Mikael is suggesting.
> 
> Ok, on it :)

Per Slack convo, we decided to defer to a follow-up PR.

**Review by simison — APPROVED** [review]
This is looking good now. 👍 

We're moving ⛰ s here!

File: packages/jetpack-blocks/.eslintrc.js (hunk 1 above)
Comment (simison) [reply packages/jetpack-blocks/.eslintrc.js]: Looks like these configs were also in overrides file:

https://github.com/Automattic/wp-calypso/blob/8f4f95a70f5327175c54dddb2fe7b8057ee0593e/.eslintrc.js#L24-L46

...so I removed the config file in https://github.com/Automattic/wp-calypso/pull/31759/commits/eca830cae495502d6e7ed082c55732f567acad50

**Comment by scruffian** [comment]
> Per Slack convo, we decided to defer to a follow-up PR.

Done in https://github.com/Automattic/wp-calypso/pull/31800

**Pull request merged**